src/elements/content-sidebar/activity-feed/__tests__/Avatar.test.js

```diff
@@ -13,16 +13,6 @@ describe('elements/content-sidebar/ActivityFeed/Avatar', () => {
 
     const getWrapper = props => shallow(<Avatar {...props} />);
 
-    test('should render nothing if no avatarUrl in state and getAvatarUrl method was passed', () => {
-        expect(
-            getWrapper({ user, getAvatarUrl })
-                .dive()
-                .find('Avatar')
-                .exists(),
-        ).toBe(false);
-        expect(getAvatarUrl).toBeCalledWith(user.id);
-    });
```

Comment:
this test should've been removed in this PR: https://github.com/box/box-ui-elements/pull/654

when this block was removed:
```
if (!avatarUrl) {
    return null;
}
```

the Avatar component is always rendered now and this test was unintentionally passing because of the `dive()`